src/components/MissingDocAlert.tsx

```diff
@@ -60,7 +70,7 @@ export const MissingDocAlert: React.FC<MissingDocAlertProps> = ({ me }) => {
                         <Box height={{ base: "35px", md: "50px", lg: "70px" }}>
                             <InfoIcon />
                         </Box>
-                        <Box w="60%">
+                        <Box w="90%">
```

Comment:
Just a note, using percentages like these generally just us results that are not mobile responsive. I can test this later

Reply:
oh okay, i just changed the width because the formatting felt a bit wonky but i didn't really take mobile into account, i'll put it back to 60!

Reply:
oh i meant just using percentages that are not "100%" in general.

Reply:
@aamyli You can revert it back to 90% if you think it's appropriate. As long as you've tested how it looks in both web and mobile view, it's good to ship.